p2p/stream/common/requestmanager/interface_test.go

```diff
@@ -118,7 +119,7 @@ func (st *testStream) FailedTimes() int {
 	return 0
 }
 
-func (st *testStream) AddFailedTimes() {
+func (st *testStream) AddFailedTimes(faultRecoveryThreshold time.Duration) {
```

Comment:
test will be developed in a separate PR ?

Reply:
can be another PR